app/src/main/java/com/hedvig/app/feature/insurance/ui/Subheading.kt

```diff
@@ -1,20 +1,28 @@
 package com.hedvig.app.feature.insurance.ui
 
+import androidx.compose.foundation.Canvas
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
 import androidx.compose.foundation.layout.fillMaxWidth
 import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import com.hedvig.app.ui.compose.theme.HedvigTheme
 
 @Composable
-fun Subheading(text: String) {
-    Text(
-        text = text,
-        style = MaterialTheme.typography.h6,
+fun Subheading(
```

Comment:
I think it makes sense to create some kind of `NotificationSubheading`, instead of adding more functionality to this composable. Imagine when we want to add `showIcon: Boolean`, `showStrikethrough: Boolean` etc. - this composable will contain too much logic.

Perhaps you could even wrap `Subheading`  in `NotificationSubheading`.